pyface/ui/qt4/tasks/editor_area_pane.py

```diff
@@ -82,6 +100,14 @@ def destroy(self):
         for editor in self.editors:
             self.remove_editor(editor)
 
+        if self.control is not None:
+            self.control.currentChanged.disconnect(self._update_active_editor)
+            self.control.tabCloseRequested.disconnect(self._close_requested)
```

Comment:
They can be easily disconnected without adding to the list. And since in other panes the list was used for shortcut connections only, I left the list here to be for shortcut connections only as well.

Reply:
Added them to the list